.changelog/4948.yml

```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Fixed an issue when creating a release note for an ApiModule also created release notes for deprecated integrations.
```

Comment:
Fixed an issue when creating a release note for an ApiModule, also created release notes for deprecated integrations.